curriculum/challenges/english/10-coding-interview-prep/the-odin-project/working-with-text-question-e.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 637f4e7972c65bc8e73dfe2b
+videoId: gW6cBZLUk6M
+title: Working With Text Question E
+challengeType: 11
+dashedName: working-with-text-question-e
+---
+# --description--
+
+You may have noticed that in all the examples in this lesson you indent any elements that are within other elements. This is known as nesting elements.
+
+When you nest elements within other elements, you create a parent and child relationship between them. The nested elements are the children and the element they are nested within is the parent.
+
+In the following example, the body element is the parent and the paragraph is the child:
+
+```html
+<html>
+  <head>
+  </head>
+  <body>
+    <p>Lorem ipsum dolor sit amet.</p>
+  </body>
+ </html>
+```
+
+Just as in human relationships, HTML parent elements can have many children. Elements at the same level of nesting are considered to be siblings.
+
+For example, the two paragraphs in the following code are siblings, since they are both children of the body tag and are at the same level of nesting as each other:
```

Comment:
```suggestion
For example, the two `p` elements in the following code are siblings, since they are both children of the `body` tag and are at the same level of nesting as each other:
```